PayloadPkg/FirmwareUpdate/FirmwareUpdate.inf

```diff
@@ -22,6 +22,7 @@
 
 [Sources]
   FirmwareUpdate.c
+  InternalFirmwareUpdate.c

```

Comment:
It is not clear what is inside InternalFirmwareUpdate.c and what this file is used for from the name.
From the function, all functions are related with bios region or cfg region update.  Not sure if container update will be support in this file. If yes, maybe rename it to BiosFwUpdate.c

Reply:
This file is not related to anything specific, going forward I need to add several more functions to support container and component updates and firmware update.c file is getting big. so I split all update routines and only firmware update internal functions to separate file

Reply:
Let us call it  FirmwareUpdateHelper.c